storage/url/url_test.go

```diff
@@ -432,3 +433,137 @@ func TestURLWithMode(t *testing.T) {
 		}
 	}
 }
+
+func TestURLSetRelative(t *testing.T) {
+	type testcase struct {
+		name   string
+		base   string
+		target string
+		expect string
+	}
+	var tests []testcase
+	if runtime.GOOS == "windows" {
```

Comment:
Is this really necessary? We already have path separator conversion for Windows.

Reply:
If the tests cases are the same, please try to use 1 scenario for both operation systems using filepath separator.